tools/benchmarking/benchmark_params.yaml

```diff
@@ -0,0 +1,30 @@
+seed: 42
+writer:
+  - csv
+  - tensorboard
+grid_search:
+  dataset:
+    category:
+      - bottle
+      - cable
+      - capsule
+      - carpet
+      - grid
+      - hazelnut
+      - leather
+      - metal_nut
+      - pill
+      - screw
+      - tile
+      - toothbrush
+      - transistor
+      - wood
+      - zipper
+    image_size: [224]
+  model_name:
+    - padim
+    - patchcore
+  model:
+    normalization_method:
```

Comment:
Not sure if we want this here by default. It will add a lot of time to the grid search, and rperformance between the different normalization methods should be the same.